auth.go

```diff
@@ -32,7 +33,29 @@ func V2ValidateSequenceAccountProof() ethauth.ValidatorFunc {
 }
 
 func ValidateSequenceAccountProof() ethauth.ValidatorFunc {
-	return V2ValidateSequenceAccountProof()
+	eip6492Validator := EIP6492ValidateSignature()
+	legacyValidator := V2ValidateSequenceAccountProof()
+
+	return func(ctx context.Context, provider *ethrpc.Provider, chainID *big.Int, proof *ethauth.Proof) (bool, string, error) {
+		// NOTICE: We only need the legacy validator so we can deploy these API updates
+		// without having to pause the wallet, otherwise we need to coordinate both.
+		// as soon as we have EIP-6492 running on prod we can remove this.
+		eip6492Valid, _, _ := eip6492Validator(ctx, provider, chainID, proof)
+		if eip6492Valid {
+			return true, proof.Address, nil
+		}
+
+		legacyIsValid, addr, err := legacyValidator(ctx, provider, chainID, proof)
+		if err != nil {
+			return false, addr, err
+		}
+
+		if legacyIsValid {
+			fmt.Printf("WARNING: Legacy signature validation used, please upgrade to EIP-6492: %s, %s\n", proof.Address, proof.Signature)
```

Comment:
Indeed -- if we print this to just stdout like this, it will not be picked up by our logger.

You have to pass the structured logger around. The way we support it in go-sequence is we have a `logger.Logger` interface which uses github.com/goware/logger and then we pass the logger around, when running, it will log to the structured logger.

I've pushed a commit to fix this